File: chap4/NombreDea.ml
(* Ces programmes sont sous licence CeCILL-B V1. *)

let nombreDea chaine =
   let nombre = ref 0 in
   for i = 0 to String.length chaine - 1 do
     if chaine.[i] = 'a' then
         nombre := !nombre + 1;
   done;
   !nombre

let _ =
  print_int ( nombreDea "abracadabra");
  print_newline ()

File: chap4/NombreDea.js
// Ces programmes sont sous licence CeCILL-B V1.
function nombreDea (s) {
var i, n;
n = 0;
for (i = 0; i <= s.length - 1;i = i + 1) {
	if (s.charAt(i)=="a") {
		n = n + 1;
	}
}
return n;
}
ISNprintln(nombreDea("abracadabra"));
